great_expectations/data_context/store/data_context_store.py

```diff
@@ -23,6 +23,7 @@ class DataContextStore(ConfigurationStore):
         DataContextVariableSchema.DATASOURCES,
         DataContextVariableSchema.EVALUATION_PARAMETER_STORE_NAME,
         DataContextVariableSchema.EXPECTATIONS_STORE_NAME,
+        DataContextVariableSchema.INCLUDE_RENDERED_CONTENT,
```

Comment:
We want to exclude this from the payload sent to Cloud? Just confirming

Reply:
Wow nice catch! No this was not intended. ✅